data.sql

```diff
@@ -1,5 +1,13 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animals (name; date_of_birth; escape_attempts; neutered; weight_kg)
+ VALUES ('Agumon'; '2020-02-03'; 0; 'true'; 10.23),
+
+INSERT INTO animals (name; date_of_birth, escape_attempts; neutered; weight_kg)
+ VALUES ('Gabumon'; '2018-11-15'; 2; 'true'; 8),
+
+INSERT INTO animals (name; date_of_birth; escape_attempts; neutered; weight_kg) 
+VALUES ('Pikachu'; '2021-01-07'; 1; 'false'; 15.04),
+
+INSERT INTO animals (name; date_of_birth; escape_attempts; neutered; weight_kg) 
+VALUES ('Devimon'; '2017-05-12'; 5; 'true', 11);
```

Comment:
- [ ] You mis-understood the requested changes by the previous reviewer, every sql statement ends with `;`. This what tells the query language that nothing is expected again but to execute the statement. While listing the data types and their values, `,` is used to seperate them, not `;`. Kindly use `,` to seperate your data type and `;` to end every statement. Example. :point_down: 

```
   NSERT INTO animals (name, date_of_birth, escape_attempts, 
     neutered, weight_kg) 
    VALUES ('Devimon', '2017-05-12', 5, 'true', 11);
```

Reply:
Thank you so much for carefully making me understand and implement the required changes, kindly review it again. @Qoosim